components/PedometerScreen.js

```diff
@@ -0,0 +1,62 @@
+import React, { useState, useEffect } from "react";
+import { Pedometer } from "expo-sensors";
+import styled from "styled-components/native";
+
+const Container = styled.View`
+  flex: 1;
+  justify-content: center;
+`;
+
+const Title = styled.Text`
+  font-size: 40px;
+  color: palevioletred;
+  margin: 10px;
+  text-align: center;
+`;
+
+const Text = styled.Text`
+  font-size: 30px;
+  color: black;
+  text-align: center;
+  margin-top: 10px;
+`;
+
+const PedometerScreen = () => {
+  const [pedometer, setpedometer] = useState("");
+  const [stepCount, setStepCount] = useState(0);
+
+  const Distance = (stepCount / 1300).toFixed(3);
+
+  useEffect(() => {
+    subscribe();
+    return () => _unsubscribe();
+  }, []);
+  const _unsubscribe = () => {
+    subscription && _subscription.remove();
+    subscription = null;
+  };
+  const subscribe = () => {
+    const subscription = Pedometer.watchStepCount((result) => {
```

Comment:
On the useState pedometer on line 25 is not use anywhere when im looking on your code in VSC

Reply:
pedometer is for checking whether a pedometer is available on the device. It shows true on mobile and false in web browsers. I did not add it because it just shows a boolean in text